portfolio/src/main/webapp/script.js

```diff
@@ -61,3 +61,171 @@ function addRandomFunFact() {
 
 }
 
+/* 
+ * Creates the pagination bar given the number of pages and the previous active page number.
+ * Returns the current active page number.
+ */
+function createPagination(numPages, activePageNumber) {
+  // If the previous active page number is greater than the current number of pages, then set it to 1;
+  // Otherwise, use the previous as current active page number.
```

Comment:
This is pretty straightforward and basically repeats what the code does.
You want to talk about the intention behind why you're doing something.
Eg (not sure if this is accurate, but to give you an idea): "Reset the active page number if user attempts to go beyond the number of available pages" or "beyond the last page" .. Something along those lines

Reply:
Got it!